pydata/templates/pydata/generic_import_form.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base_nav_fixed.html" %}
```

Comment:
This template would be used at the `/model/add/` view that is only concerned with the import of a model instance.
A similar template would exist for `/model/<slug>/edit` view that would allow both imports and edits.

I haven't added the functionality to edit the existing PyData events, persons and tasks before we get a response from Leah or Andy.